bodhi/server/templates/update.html

```diff
@@ -575,10 +575,14 @@ <h1>
       % endif
 
     </h1>
-    <h4>
+    <h4 id='shortdesc'>
       ${update.type} update
-      in ${update.release.long_name}
-      for ${update.beautify_title(amp=True) | n}
+      in ${update.release.long_name} for
+      <a href="${request.route_url('updates') + '?packages=' + build.package.name}">
+        ${update.beautify_title(amp=True) | n}
```

Comment:
There isn't a variable called ```build``` in this scope, which leads to a traceback:

```
Traceback (most recent call last): 
  File "/usr/lib/python2.7/site-packages/pyramid_mako/__init__.py", line 148, in __call__ 
    result = template.render_unicode(**system) 
  File "/usr/lib/python2.7/site-packages/mako/template.py", line 471, in render_unicode 
    as_unicode=True) 
  File "/usr/lib/python2.7/site-packages/mako/runtime.py", line 838, in _render 
    **_kwargs_for_callable(callable_, data)) 
  File "/usr/lib/python2.7/site-packages/mako/runtime.py", line 873, in _render_context 
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs) 
  File "/usr/lib/python2.7/site-packages/mako/runtime.py", line 899, in _exec_template 
    callable_(context, *args, **kwargs) 
  File "/home/vagrant/bodhi/bodhi/server/templates/master.html", line 188, in render_body 
    ${self.body()} 
  File "/home/vagrant/bodhi/bodhi/server/templates/update.html", line 581, in render_body 
    <a href="${request.route_url('updates') + '?packages=' + build.package.name}"> 
UnboundLocalError: local variable 'build' referenced before assignment
```

Though many updates only reference a single build, updates can contain multiple builds.

The Builds tab already contains this for each of the builds. Does that not satisfy what you are trying to achieve?

Reply:
Builds tab redirects to https://koji.fedoraproject.org/koji/buildinfo?buildID=1044924

How to see which variables exist at the given scope?